### PR TITLE
Kpp added properly

### PR DESCRIPTION
**Pull request by dhruvbalwada**
Added KPP vars metadata into a new dictionary. This dictionary can potentially be expanded in the future with other such package variables. 
Tests have not been added for these variables, will be subject of a future PR.

### CODE REVIEW

File: xmitgcm/mds_store.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
     horizontal_coordinates_spherical, horizontal_coordinates_cartesian, \
     horizontal_coordinates_curvcart, \
     vertical_coordinates, horizontal_grid_variables, vertical_grid_variables, \
-    volume_grid_variables, state_variables, aliases
+    volume_grid_variables, state_variables, aliases, package_state_variables
```
Comment (rabernat) [inline xmitgcm/mds_store.py]: I love it!

**Comment by rabernat** [comment]
Don't worry about the failing test. That will be fixed by #54.

**Comment by rabernat** [comment]
I am inclined to merge this. @roxyboy, this is how you would add the Darwin package variables.

This fixes #49

File: xmitgcm/mds_store.py (hunk 1 above)
Comment (dhruvbalwada) [reply xmitgcm/mds_store.py]: My first PR. :')

**Pull request merged**